### ISSUE TITLE
Add config for linting astro files w/o passing `--ext`

### ISSUE THREAD

**Issue by jasikpark** [ota-meshi/eslint-plugin-astro#34]
If you add `*.astro` to the `overrides.files` config value, you can lint astro (and other file-endings like ts) w/o needing an extension flag:

https://eslint.org/docs/user-guide/migrating-to-7.0.0#-lint-files-matched-by-overridesfiles-by-default

ex, the current config I'm working on, derived from config written by @IanVS:

```js
overrides: [
    // By default eslint only lints .js.  Adding others to overrides will cause them to lint as well
    // https://eslint.org/docs/user-guide/migrating-to-7.0.0#lint-files-matched-by-overridesfiles-by-default
    { files: ["*.ts", "*.tsx", "*.jsx"] },
    // All .cjs & .cts files and eslint rules are commonjs, which means they're processed by node and can use require()
    {
      files: ["*.cjs", "*.cts", "eslint-rules/*.js"],
      env: { node: true },
      rules: {
        "@typescript-eslint/no-var-requires": "off",
        "@typescript-eslint/no-require-imports": "off",
        "import/no-nodejs-modules": "off",
      },
    },
    {
      files: ["*.astro"],
      parser: "astro-eslint-parser",
      parserOptions: {
        parser: "@typescript-eslint/parser",
        extraFileExtensions: [".astro"],
      },
    },
  ],
  ```

this should lint all typescript and astro files, assuming all of the plugins and whatnot are configured of course

**TLDR: Reccomend specifying `files: ["*.astro"],` to support linting astro, rather than requiring passing `--ext .astro` to eslint**

**Comment by ota-meshi** [comment]
Thank you for posting this issue!

The shareable configuration of this plugin provides `files: ["*. astro"]`, so in practice most users will probably work without the `--ext` option.  
https://github.com/ota-meshi/eslint-plugin-astro/blob/1faa162c151052b037ce19dbafc0bbd66b44622f/src/configs/base.ts#L6

However, for example, users who use `plugin:typescript-eslint/base` need `--ext` because they don't have `files: ["*.ts"]`.
Users may not need to use `--ext` if they are aware of the contents of the shareable configuration they have used.
However, to avoid confusion for some users who don't know what configuration they are using, I wrote the documentation to use `--ext`.
https://github.com/ota-meshi/eslint-plugin-astro#running-eslint-from-the-command-line

We probably need to briefly explain what you can omit `--ext` in the documentation. But I can't explain it well 😓
